addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -739,12 +747,54 @@ QUnit.test('use a command', async function (assert) {
     );
 });
 
+QUnit.test('no command should be displayed for this channel type', async function (assert) {
+    assert.expect(3);
+
+    this.data['mail.channel'].records.push({ channel_type: 'chat', id: 20 });
+    await this.start();
+    const thread = this.env.models['mail.thread'].findFromIdentifyingData({
+        id: 20,
+        model: 'mail.channel',
+    });
+    await this.createComposerComponent(thread.composer);
+
+    assert.containsNone(
+        document.body,
+        '.o_ComposerSuggestionList_list',
+        "command suggestions list should not be present"
+    );
+    assert.strictEqual(
+        document.querySelector(`.o_ComposerTextInput_textarea`).value,
+        "",
+        "text content of composer should be empty initially"
+    );
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/");
+    });
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`)
+            .dispatchEvent(new window.KeyboardEvent('keydown'));
+        document.querySelector(`.o_ComposerTextInput_textarea`)
+            .dispatchEvent(new window.KeyboardEvent('keyup'));
+    });
+    assert.containsNone(
+        document.body,
+        '.o_ComposerSuggestion',
+        "should not have a command suggestion because of the specific channel type 'chat'"
```

Comment:
_"should not prompt (command) suggestion after typing "/" (reason: no channel commands in chat channels)"_